lib/app.js

```diff
@@ -33,7 +33,7 @@ function App(config, finalizer) {
 
   this.reportFileName = this.config.get('report_file');
   this.reportFileStream = this.initReportFileStream(this.reportFileName);
-  this.reporter = this.initReporter(this.config.get('reporter'), this.reportFileStream);
+  this.reporter = this.initReporter((this.config.get('reporter') || 'tap'), this.reportFileStream);
```

Comment:
This should be required as `tap` is still the default https://github.com/testem/testem/blob/master/lib/config.js#L160.